src/components/DriverCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Rating from './Rating'
+
+function DriverCard({ name, rating, img, car }) {
+
+  const fontSize = { fontSize: 30 }
+
+  const divStyle = {
+    width: 1300,
+    height: 250,
+    color: 'white',
+    backgroundColor: 'blue',
+    borderRadius: 15,
+    marginBottom: 50,
+  }
+
+  return (
+    <div style={divStyle} className="d-flex  align-items-center justify-content-center">
+      <img src={img} alt="img" width='180' height='180' className='rounded-circle' />
+      <div className='mx-5 d-flex flex-column '>
+        <p style={fontSize}><b>{name}</b></p>
+        <p style={fontSize}><Rating children={rating} /></p>
```

Comment:
```suggestion
        <p style={fontSize}><Rating>{rating}</Rating></p>
```

El children se pasa asi, la otra forma se puede, pero la manera correcta es esta vale?